src/com/facebook/buck/apple/ApplePackageDescription.java

```diff
@@ -105,10 +105,19 @@ public ApplePackageDescription(
           applePackageConfigAndPlatformInfo.get(),
           new BuildTargetSourcePath(bundle.getBuildTarget()));
     } else {
+      Set<Flavor> platformFlavors = getPlatformFlavorsOrDefault(params.getBuildTarget());
+      AppleCxxPlatform platform = null;
+
+      for (Flavor flavor : platformFlavors) {
```

Comment:
Actually `Iterables.tryFind()` or `FluentIterable.from().firstMatch()` should be better.